### PR TITLE
Add description field to user

### PR DESCRIPTION
**Pull request by nhamhung**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/edit/EditUserCommand.java [hunk 1]
```diff
@@ -74,23 +78,28 @@ public CommandResult execute(Model model) throws CommandException {
...
+            throw new CommandException(MESSAGE_FROM_TO_MISORDER);
+        }
+
         model.setUser(editedUser);
         model.commitResumeBook();
 
         return new EditUserCommandResult(editedUser.toString(),
-                String.format(MESSAGE_EDIT_PERSON_SUCCESS, editedUser),
-                model.getDisplayType());
+                MESSAGE_EDIT_PERSON_SUCCESS, "");
```
Comment (helloImHai) [inline src/main/java/seedu/address/logic/commands/edit/EditUserCommand.java]: It's not safe to not pass up model.getDisplayType()

File: src/main/java/seedu/address/model/item/field/Time.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
 
     public static final String MESSAGE_CONSTRAINTS = "Time should be in the format MM-yyyy";
     public static final String VALIDATION_REGEX = "^(1[0-2]|0[1-9])-[0-9]{4}$";
+    public static final String MESSAGE_FROM_TO_MISORDER = "\"from\" field cannot be after \"to\" field. ";
```
Comment (helloImHai) [inline src/main/java/seedu/address/model/item/field/Time.java]: Same comment that Duong made on mine lol, "\'from\' field cannot be later \'to\' field.". It's cuz the word "after" can be confusing

**Review by helloImHai — CHANGES_REQUESTED** [review]
Nice one bro 🥇
1 small change and 1 breaking change, can you take a look?

File: src/main/java/seedu/address/logic/commands/edit/EditUserCommand.java (hunk 1 above)
Comment (nhamhung) [reply src/main/java/seedu/address/logic/commands/edit/EditUserCommand.java]: Thanks bruh :)

**Review by helloImHai — APPROVED** [review]
As it should be m8 💯💯💯

**Pull request merged**